SSA/Projects/InstCombine/ForStd.lean

```diff
@@ -131,4 +131,11 @@ theorem bitvec_minus_one : BitVec.ofInt w (Int.negSucc 0) = (-1 : _root_.BitVec
     rw [zeroBitwidth]
     simp
 
+theorem bitvec_minus_one' : BitVec.ofInt w (-1) = (-1 : BitVec w) := by
+  have x : -1 = Int.negSucc 0 := by simp
+  rw [x, bitvec_minus_one]
+
+
+
+
```

Comment:
```suggestion
theorem bitvec_minus_one' : BitVec.ofInt w (-1) = (-1 : BitVec w) := by
  have x : -1 = Int.negSucc 0 := by simp
  rw [x, bitvec_minus_one]
  
```

Reply:
@bollu, what is the change you are suggesting here?

Reply:
@tobiasgrosser aha,  I wanted to nuke the extra newlines, but it seems like github's "suggestion" algorithm strips trailing newline, so it looks like I'm suggesting we add a newline..

Reply:
Done.